examples/UpdateContract/Program.cs

```diff
@@ -0,0 +1,94 @@
+using CommandLine;
+using Concordium.Sdk.Client;
+using Concordium.Sdk.Types;
+using Concordium.Sdk.Wallets;
+
+namespace Transactions.UpdateContract;
+
+// We disable these warnings since CommandLine needs to set properties in options
+// but we don't want to give default values.
+#pragma warning disable CS8618
+
+internal sealed class UpdateTransactionExampleOptions
+{
+    [Option(
+        'k',
+        "keys",
+        HelpText = "Path to a file with contents that is in the Concordium browser wallet key export format.",
+        Required = true
+    )]
+    public string WalletKeysFile { get; set; }
+    [Option(HelpText = "URL representing the endpoint where the gRPC V2 API is served.",
+        Default = "http://node.testnet.concordium.com:20000/")]
```

Comment:
This URL will be deprecated soon, best to use https://grpc.testnet.concordium.com:20000 instead.

Reply:
I have updated all of the other examples as well